config/dwdirectiverules/dw_directive_rules.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: dws.cray.hpe.com/v1alpha1
+kind: DWDirectiveRule
+metadata:
+  name: dws-test-driver-directive-rules
+spec:
+  - command: Proposal
+    watchStates: Proposal
+    driverLabel: tester
+    ruleDefs:
+      - key: action
+        type: string
+        isRequired: true
+        isValueRequired: true
+      - key: message
+        type: string
+        isValueRequired: true
+  - command: Setup
+    watchStates: Setup
+    driverLabel: tester
+    ruleDefs:
+      - key: action
+        type: string
+        isRequired: true
+        isValueRequired: true
+      - key: message
+        type: string
+        isValueRequired: true
+  - command: DataIn
+    watchStates: DataIn
+    driverLabel: tester
+    ruleDefs:
+      - key: action
+        type: string
+        isRequired: true
+        isValueRequired: true
+      - key: message
+        type: string
+        isValueRequired: true
+  - command: PreRun
+    watchStates: PreRun
+    driverLabel: tester
+    ruleDefs:
+      - key: action
+        type: string
+        isRequired: true
+        isValueRequired: true
+      - key: message
+        type: string
+        isValueRequired: true
+  - command: PostRun
+    watchStates: PostRun
+    driverLabel: tester
+    ruleDefs:
+      - key: action
+        type: string
+        isRequired: true
+        isValueRequired: true
+      - key: message
+        type: string
+        isValueRequired: true
+  - command: DataOut
+    watchStates: DataOut
+    driverLabel: tester
+    ruleDefs:
+      - key: action
+        type: string
+        isRequired: true
+        isValueRequired: true
+      - key: message
+        type: string
+        isValueRequired: true
+  - command: Teardown
```

Comment:
Add ability to add "hurry" flag.

Reply:
Will the conduit driver be able to set the hurry flag? I thought that the hurry flag was set from the workload manager.

Reply:
Conduit will not be able to set the hurry flag.  I image an admin or user of Slurm would have to specify it through the Slurm GUI/API.